lib/logstash/outputs/datadog_logs.rb

```diff
@@ -179,6 +185,14 @@ def close
   class DatadogHTTPClient < DatadogClient
     require "manticore"
 
+    RETRYABLE_EXCEPTIONS = [
+        ::Manticore::Timeout,
+        ::Manticore::SocketException,
+        ::Manticore::ClientProtocolException,
+        ::Manticore::ResolutionFailure,
+        ::Manticore::SocketTimeout
```

Comment:
technically you can remove `SocketTimeout` since it's a subclass of `Timeout` (see https://www.rubydoc.info/github/cheald/manticore/Manticore/Timeout)

Reply:
Correct, updated